load-testing/tests/relays_stress_test.go

```diff
@@ -430,11 +390,11 @@ func (s *relaysSuite) MoreActorsAreStakedAsFollows(table gocuke.DataTable) {
 	gateways := fundedGateways[:s.gatewayInitialCount]
 	applications := fundedApplications[:s.appInitialCount]
 
+	stakedActors := append(suppliers, gateways...)
```

Comment:
Trying to be consistent with "funded" above.

```suggestion
	stakedActors := append(stakedSuppliers, stakedGateways...)
```